app/src/main/java/fr/free/nrw/commons/upload/FileUtils.java

```diff
@@ -164,7 +164,7 @@ public static boolean recursivelyCreateDirs(String dirPath) {
     }
 
     /**
-     * Check if file exists in local dirs
+     * Check if file destroyed in local dirs
```

Comment:
I think you may need to revert most of these "Find and replace" accidents ;)

Reply:
oh wow, I just wanted to revert a var name (exists) to destroyed... surely will revert them